### ISSUE TITLE
MongoDB performance dashboards errors

### ISSUE THREAD

**Issue by vietwow** [elastic/packetbeat-dashboards#88]
Hi,

 I use beats 1.1.1 (with ES 2.2 and Kibana 4.4.1). When I import beats-dashboards 1.1.1, and access Kibana web -> Setting tab -> Indices tab -> Index Patterns, click on "packetbeat-*" and it warning :

> Mapping conflict! A field is defined as several types (string, integer, etc) across the indices that match this pattern. You may still be able to use these conflict fields in parts of Kibana, but they will be unavailable for functions that require Kibana to know their type. Correcting this issue will require reindexing your data.

So when I use packetbeat to monitor mogodb, and see the dashboard "MongoDB performance", it corrupts 3 dashboards (remainning other dashboards still is working well) with the error :

> Could not locate that index-pattern-field (id: resource)

I suspect the reason is the Kibana Index Pattern of dashboard "MongoDB Performance" is old version, so it not updated yet to compatible with PacketBeat index template of Elasticsearch ?

Best Regards,
VietNC

**Comment by maziyarpanahi** [comment]
@vietwow yes I have the same issue. Some of the visualizations are using fields that they are no longer exist apparently.

**Comment by gsaslis** [comment]
anyone ever found their way out of this one?

**Comment by ruflin** [comment]
Dashboards are now part of the beats repo directly found here: https://github.com/elastic/beats Which version of the dashboards did you use with which elastic stack versions?
